crates/bevy_gltf/src/loader.rs

```diff
@@ -714,9 +710,8 @@ fn load_node(
 ) -> Result<(), GltfError> {
     let transform = gltf_node.transform();
     let mut gltf_error = None;
-    let mut node = world_builder.spawn(SpatialBundle::from(Transform::from_matrix(
-        Mat4::from_cols_array_2d(&transform.matrix()),
-    )));
+    let transform = Transform::from_matrix(Mat4::from_cols_array_2d(&transform.matrix()));
```

Comment:
What's the purpose of Transform -> Matrix -> Transform?

Reply:
Those are two different Transform types: one is the bevy Transform type and the other is the glTF specific transform type. I just extracted the bevy Transform into a local variable because I need it again for the Camera3dBundle if the node is a camera.

Reply:
Then can we mark them with what kind of transform each one is in the variable declaration?
i.e.
```rust
let transform: GltfTransform = ...;
let transform: Transform = ...;
```